<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Timer_Config</class>
 <widget class="QWidget" name="Timer_Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>311</width>
     <height>550</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10%;
background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>65</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MaoKenTangYuan (beta)</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>透明度:</string>
    </property>
   </widget>
   <widget class="QLabel" name="transpearnt_pctg">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MaoKenTangYuan (beta)</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>65</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MaoKenTangYuan (beta)</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>460</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MaoKenTangYuan (beta)</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MaoKenTangYuan (beta)</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator {width: 28px; height: 28px}

QCheckBox::indicator:checked {image: url(:/icon/checked.png);}

QCheckBox::indicator:enabled:unchecked {
        image: url(:/icon/unchecked.png);
}</string>
    </property>
    <property name="text">
     <string>窗口置顶</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MaoKenTangYuan (beta)</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(212, 212, 212);
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(51, 222, 255, 170), stop:1 rgba(112, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string>选择背景颜色</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MaoKenTangYuan (beta)</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>自定义倒计时内容：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MaoKenTangYuan (beta)</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>在此处编辑文本</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MaoKenTangYuan (beta)</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>高考倒计时 v2.2       项目源代码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>515</y>
      <width>53</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MaoKenTangYuan (beta)</family>
      <pointsize>10</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>&lt;a href=&quot;https://github.com/xiefujiang/Timer&quot;&gt;Github&lt;/a&gt;</string>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MaoKenTangYuan (beta)</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2024</year>
      <month>6</month>
      <day>7</day>
     </datetime>
    </property>
    <property name="time">
     <time>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
     </time>
    </property>
    <property name="maximumDateTime">
     <datetime>
      <hour>23</hour>
      <minute>59</minute>
      <second>59</second>
      <year>6666</year>
      <month>6</month>
      <day>7</day>
     </datetime>
    </property>
    <property name="maximumDate">
     <date>
      <year>6666</year>
      <month>6</month>
      <day>7</day>
     </date>
    </property>
    <property name="maximumTime">
     <time>
      <hour>23</hour>
      <minute>59</minute>
      <second>59</second>
     </time>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::DaySection</enum>
    </property>
    <property name="calendarPopup">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MaoKenTangYuan (beta)</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>目标日期：</string>
    </property>
   </widget>
   <zorder>horizontalSlider</zorder>
   <zorder>transpearnt_pctg</zorder>
   <zorder>label</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>checkBox</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>label_2</zorder>
   <zorder>textEdit</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>dateEdit</zorder>
   <zorder>label_5</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
